pkg/tasks/python.go

```diff
@@ -27,13 +28,54 @@ func parserPython(config *taskConfig, task *Task) error {
 	task.featureName = "python"
 	task.featureParam = version
 
+	task.addAction(&pyenvPath{})
+	task.addAction(&pyenv{})
 	task.addAction(&pythonPyenv{version: version})
 	task.addAction(&pythonInstallVenv{version: version})
 	task.addAction(&pythonCreateVenv{version: version})
 
 	return nil
 }
 
+type pyenvPath struct{}
+
+func (p *pyenvPath) description() string {
+	return ""
+}
+
+func (p *pyenvPath) needed(ctx *context) (bool, error) {
+	match, _ := regexp.MatchString("\\.pyenv/bin", ctx.env.Get("PATH"))
+	return !match, nil
+}
+
+func (p *pyenvPath) run(ctx *context) error {
+	ctx.ui.TaskWarning("The PATH environment variable should have 'pyenv' in it. Try adding the following to ~/.bashrc\n\nexport PATH=\"~/.pyenv/bin:$PATH\"")
```

Comment:
```suggestion
	ctx.ui.TaskWarning("The PATH environment variable should have 'pyenv' in it. Try adding the following to ~/.bashrc\n\nexport PATH=\"~/.pyenv/shims:$PATH\"")
```

Or we could instruct to follow the steps on https://github.com/pyenv/pyenv#advanced-configuration

Reply:
Removed this task since homebrew will set up the path for you.